web/app.fluidity.money/app/queries/useUserTransactionCount.ts

```diff
@@ -64,23 +81,26 @@ export type UserTransactionCountRes = {
 
 const useUserTransactionByAddressCount = async (
   network: string,
-  address: string
+  address: string,
+  useMoralis = true
```

Comment:
See other comments. (Sorry about this being exhaustive)
Chuck a defaults constant somewhere in the app.